src/grantRegistry.ts

```diff
@@ -44,10 +47,13 @@ export function handleGrantUpdated(event: GrantUpdated): void {
     entity.id = event.params.id.toHex()
     entity.owner = event.params.owner
     entity.payee = event.params.payee
-    entity.metaPtr = event.params.metaPtr
+
+    const metaPtr = event.params.metaPtr
+    entity.metaPtr = [metaPtr.protocol.toString(), metaPtr.pointer].join('-')
```

Comment:
Why are we returning this as a string instead of an object like this:
```suggestion
    entity.metaPtr = { protocol: metaPtr.protocol.toString(), pointer: metaPtr.pointer }
```

(We do this in other places too, but I just only left a comment here)

Reply:
We had troubles getting the typings to work in `schema.graphql` - we wanted to create a `Type` for `MetaPtr`, but when we ran `codegen` the build failed telling us that we had to make it a directive and add an `ID`.

This adds an extra step on the dGrants side, but it gets us closer now, if we need to iterate on this then I'm happy to spend some time working out the right way to do it - but we wanted to get this in sooner rather than later.

Reply:
Yea I feel like it’d be nice for the subgraph to return the same types as the contract, so maybe we create a separate issue to track this? Might also be able to figure it out by looking at other subgraphs for contracts that return structs, such Uniswap V3

Reply:
Great suggestion, I've dug in to how it works on Uniswap V3 and have raised this PR: https://github.com/dcgtc/dgrants-subgraph/pull/6 🙌  

Thanks @mds1!!